pkg/registry/servicecatalog/broker/strategy_test.go

```diff
@@ -61,3 +94,57 @@ func TestBroker(t *testing.T) {
 		t.Fatalf("Fresh broker should have empty status")
 	}
 }
+
+func TestValidateUpdateStatusPrepareForUpdate(t *testing.T) {
+	cases := []struct {
+		name                string
+		old                 *sc.Broker
+		newer               *sc.Broker
+		shouldChecksum      bool
+		checksumShouldBeSet bool
+	}{
+		{
+			name:                "not ready -> not ready",
```

Comment:
I'm not sure what this comment means, and I doubt we do what you're asking about for any other test - seems like it shouldn't be a blocker to merging this PR, but perhaps you can clarify.